entrypoint.sh

```diff
@@ -2,6 +2,25 @@
 
 set -eu
 
+function get_diff() {
+  base="$1"
+  sha="$2"
```

Comment:
@pkit This looks good still getting one strange error when a submodule is deleted.

See: https://github.com/tj-actions/changed-files/runs/5296218109?check_suite_focus=true#step:3:89

![Screen Shot 2022-02-22 at 6 20 47 PM](https://user-images.githubusercontent.com/17484350/155236410-d42d0893-72a2-464e-92fd-0314a0d9dcdb.png)

Reply:
Hmm, will check it out

Reply:
@pkit Found the issue which was related to missing `submodule: true` which should be include in the checkout action.

Reply:
Yup, seen your commits.